src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -140,8 +141,11 @@ export class BrowsingContextImpl {
     );
 
     // No need in awaiting for target to be unblocked.
+    // noinspection ES6MissingAwait
```

Comment:
This is from IntelliJ, right? It would be nice if you could either remove this annotation OR find something IDE-agnostic to express it. If this were python I am sure `# noqa` would work. For TypeScript there must be some other way, IDE-agnostic, to express "disable linting the next line".

Reply:
I was doing some research, you can just prepend `void` to the expression, that should signal to the compiler there's no need to await for the promise.

Reply:
> Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator.eslint[@typescript-eslint/no-floating-promises](https://typescript-eslint.io/rules/no-floating-promises)